lib/ex_spirit/parser/text.ex

```diff
@@ -530,13 +531,13 @@ defmodule ExSpirit.Parser.Text do
                 }
               parser when is_function(parser, 1) ->
                 %{context |
-                  position: context.position + byte_size(<<c::utf8>>),
+                  position: context.position,
                   column: if(c===?\n, do: 1, else: context.column+1),
                   line: context.line + if(c===?\n, do: 1, else: 0),
                 } |> parser.()
```

Comment:
So yeah, probably just turn this whole set into just `parser.(context)` or `context |> parser.()` or whatever.  >.>